packages/api/src/models/cropVarietyModel.js

```diff
@@ -83,6 +83,8 @@ class CropVariety extends BaseModel {
         crop_id: { type: 'integer' },
         farm_id: { type: 'string' },
         crop_variety_name: { type: ['string', null] },
+        crop_varietal: { type: 'string', minLength: 1, maxLength: 255 },
+        crop_cultivar: { type: 'string', minLength: 1, maxLength: 255 },
```

Comment:
It looks like in the frontend register you limit these to 200 chars?

Reply:
Yes, this came up before with your Crop_type creation flow ticket, and it was agreed that the character limit on the back end for Genus and Species fields be kept as 255 characters even though the requirement on the ticket and enforced on the front end was 200 characters. I figured this also followed for these fields

Reply:
Yeah kinda, the agreement was 200 is arbitrary but it needs to match frontend. This can stay 255 but then the frontend validation needs to change. Either way is good enough -- I tagged those spots

Reply:
Oh, I see